tests/testthat/test-validate-inputs.R

```diff
@@ -86,13 +86,15 @@ test_that("do_validate_programme validates programme file", {
   expect_equal(data$filters$age, expected_filters)
 })
 
-test_that("do_validate_anc validates ANC file", {
+test_that("do_validate_anc validates ANC file and gets data for plotting", {
```

Comment:
Here you test for existance of `prevalance` and `art_coverage` but in the schema only for `area_id` (`AncResponseData.schema.json`). Should these be harmonised?

Reply:
Yeah they should, I've updated that now.